packages/utilities/src/initials.ts

```diff
@@ -3,7 +3,8 @@
  * The first part matches characters within parenthesis, including the parenthesis.
  * The second part matches special ASCII characters except space, plus some unicode special characters.
  */
-const UNWANTED_CHARS_REGEX: RegExp = /\([^)]*\)|[\0-\u001F\!-/:-@\[-`\{-\u00BF\u0250-\u036F\uD800-\uFFFF]/g;
+// eslint-disable-next-line @fluentui/max-len
+const UNWANTED_CHARS_REGEX: RegExp = /[\(\[\{][^\)\]\}]*[\)\]\}]|[\0-\u001F\!-/:-@\[-`\{-\u00BF\u0250-\u036F\uD800-\uFFFF]/g;
```

Comment:
Hm, the regex becomes quite unreadable after a while. Can we try to split it into sub-regexes with descriptive names, and then use a combination of different regexes instead? That will make it a lot easier to understand and maintain this in the future

Reply:
Would splitting it into two for the brackets and then special characters  work, or were you thinking even more broken down then that

Reply:
Is the suggestion to move to something like this?

```
const UNWANTED_CHARS_REGEX: RegExp = /[\(\[\{][^\)\]\}]*[\)\]\}]|[\0-\u001F\!-/:-@\[-`\{-\u00BF\u0250-\u036F\uD800-\uFFFF]/g;

====

var openBrackets = "[\(\[\{][^\)\]\}]";
var notCloseBrackets = "[^\)\]\}]";
var closeBrackets = "[\)\]\}]";
var miscSpecialChars = "[\0-\u001F\!-/:-@\[-`\{-\u00BF\u0250-\u036F\uD800-\uFFFF]";

const UNWANTED_CHARS_REGEX = new RegExp(openBrackets + notCloseBrackets + "*" + closeBrackets + "|" + miscSpecialChars, "g");
```

Reply:
oh Elisabeth's new change is good too

Reply:
Yup, like the latest change! A lot easier to read :)